src/features/wallet/send/SendView.tsx

```diff
@@ -420,33 +372,43 @@ const SendView = ({ scanResult, sendType, hotspot, isSeller }: Props) => {
     }
   }
 
-  const constructTxn = async () => {
+  const constructTxns = async () => {
     if (type === 'payment') {
-      return makePaymentTxn(balanceAmount.integerBalance, address, getNonce())
+      return Promise.all(
+        sendTransfers.map(async ({ address, balanceAmount }) => {
+          return makePaymentTxn(
+            balanceAmount.integerBalance,
+            address,
+            getNonce(),
+          )
+        }),
+      )
     }
 
     if (type === 'dc_burn') {
-      return makeBurnTxn(
-        balanceAmount.integerBalance,
-        address,
-        getNonce(),
-        memo,
+      return Promise.all(
```

Comment:
Is supporting multiple burns in the same QR something you wanted to do? If so maybe we can talk about that. Submitting rapid transactions like this will probably fail and we may need to make some changes to support burns in this way. See my comment about about payments.

My suggestion would be that we only allow multiple payee / amounts for payment transactions at this moment.